README.md

```diff
@@ -58,3 +58,9 @@ cp .vscode/settings.json.default .vscode/settings.json
 ```
 
 And you should now be set up to start working in the repo! The easiest way to do this is for many tasks is to use [`bqetl`](https://mozilla.github.io/bigquery-etl/bqetl/). You may also want to read up on [common workflows](https://mozilla.github.io/bigquery-etl/cookbooks/common_workflows/).
+
+## Documentation
```

Comment:
IMO this belongs in "common workflows" in the documentation (`docs/cookbooks/common_workflows.md`) since it's more of an advanced topic.

Reply:
Agreed, will add it there 👍